zipkin/src/test/java/zipkin/internal/v2/CallTest.java

```diff
@@ -151,4 +136,83 @@ public void concurrent_executesOrSubmitsOnce() throws InterruptedException {
 
     assertThat(executeOrSubmit.get()).isEqualTo(1);
   }
+
+  @Test public void emptyList() throws Exception {
+    Call<List<String>> call = Call.emptyList();
+
+    assertThat(call.execute()).isEmpty();
+  }
+
+  @Test public void emptyList_independentInstances() throws Exception {
+    assertThat(Call.emptyList())
+      .isNotSameAs(Call.emptyList());
+  }
+
+  @Test public void flatMap_execute() throws Exception {
+    Call<String> fooCall = Call.create("foo");
+    Call<String> barCall = Call.create("bar");
+    Call<String> fooBarCall = fooCall.flatMap(foo -> barCall);
```

Comment:
Assert foo is "foo" (or could do something like `foo.equals("foo") ? barCall : badCall`)

Reply:
I added this to the first test that invokes a map or flatMap (as doing for all tests would be distracting)